greenplumpython/experimental/file.py

```diff
@@ -69,11 +71,75 @@ def _from_files(_, files: list[str], parser: NormalFunction, db: gp.Database) ->
                 ORDER BY id;
                 """
             )
+
+
+@classmethod
+def _from_files(_, files: list[str], parser: NormalFunction, db: gp.Database) -> gp.DataFrame:
+    tmp_archive_name = f"tar_{uuid.uuid4().hex}"
+    _archive_and_upload(tmp_archive_name, files, db)
     func_sig = inspect.signature(parser.unwrap())
     result_members = get_type_hints(func_sig.return_annotation)
     return db.apply(
-        lambda: parser(_extract_files(tmp_archive_name)), expand=len(result_members) == 0
+        lambda: parser(_extract_files(tmp_archive_name, "files")),
+        expand=len(result_members) == 0,
     )
 
 
 setattr(gp.DataFrame, "from_files", _from_files)
+
+
+import subprocess as sp
+import sys
+
+
+@gp.create_function
+def _install_on_server(cache_dir: str, requirements: str) -> str:
+    cmd = [
+        sys.executable,
+        "-m",
+        "pip",
+        "install",
+        "--no-index",
+        "--requirement",
+        "/dev/stdin",
+        "--find-links",
+        cache_dir,
+    ]
+    try:
+        output = sp.check_output(cmd, text=True, stderr=sp.STDOUT, input=requirements)
```

Comment:
here, can we use something as import instead of subprocess?
```
def install(package):
    if hasattr(pip, 'main'):
        pip.main(['install', package])
    else:
        pip._internal.main(['install', package])
```